doc/METRICS.md

```diff
@@ -1,7 +1,21 @@
-AAA diameter session metrics
-================================
+# erGW-AAA
+* [DIAMETER metrics](#diameter-metrics)
+* [DIAMETER session metrics](#diameter-session-metrics)
 
-ergw_aaa exposes prometheus gauge metric for the states of the different diameter handlers
+`erGW-AAA` uses [prometheus.erl](https://github.com/deadtrickster/prometheus.erl) to implement various operation metrics.
+
+# DIAMETER metrics
+
+The following metrics exist:
+| Name                                       | Type      | Labels         | Description                        |
+|--------------------------------------------|-----------|----------------|------------------------------------|
+| aaa_sessions_total                         | gauge     | handler, state | AAA of active sessions             |
+| ergw_aaa_diameter_rate_limiter_outstanding | gauge     | name, type     | The number of outstanding requests |
+| ergw_aaa_diameter_rate_limiter_rate        | gauge     | name, type     | The number of rate requests        |
```

Comment:
https://prometheus.io/docs/practices/naming/

metric names should end with `_total`.

I also don't understand what "The number of rate requests" means. Is that the configure rate?

Reply:
> I also don't understand what "The number of rate requests" means. Is that the configure rate?

Yes, for type `limit`.